### DISCUSSION TITLE
Is there a way to ignore a contact or status@broadcast?

### DISCUSSION THREAD

**Discussion by orxaz** [d99kris/nchat#129] [Q&A]
Hi!

Thanks for your nice work.

I would like to ask you if there is a way to ignore a chat. That I want is not to enter and download/see all states from the people when I hit the status@broadcast conversation when using WhatsApp.

Thanks!

**Comment by d99kris** [comment]
Hi @orxaz - no there's no such functionality in nchat today. I believe the standard whatsapp client puts broadcast messages in their respective chats, so perhaps nchat could do that as well.

Also, nchat only uses "mute" status received for a contact on first setup. I could look into handling mute updates in run-time.

Then one could use the phone app to mute unwanted chats and configure nchat to ignore them with:
```
muted_indicate_unread=0
muted_notify_unread=0
muted_position_by_timestamp=0
```

Will update here again when/if I get a chance to look more into this.

**Reply by d99kris** [reply]
Hi again @orxaz - I've had a look at this and added functionality for nchat to put broadcast messages under the regular chat for the sending user #133 and I've also improved handling of run-time updates to muted status in #134. Additionally with #135 nchat now shows if a chat is muted in the status bar.

Now, if you have already muted a contact on whatsapp before first "setup" in nchat, it will not be aware of it. You can unmute/mute the contact again for nchat to reflect it, or perform the "setup" step in nchat. The telegram implementation does not have this limitation.

**Reply by orxaz** [reply]
Many thanks!

For my use case, this software is perfect!

Great job!
